addon/components/inline-components/table-of-contents.js

```diff
@@ -43,18 +43,15 @@ export default class TableOfContentsComponent extends Component {
   }
 
   extractOutline(node) {
+    console.log(this.config);
```

Comment:
forgotten console.log

Reply:
> Not really sure i understand this PR, why are we reverting to the old configuration?

Yes to the ones without RDF namedNodes, as inline-components cannot serialize them fully.

Reply:
but wasn't the `SAY('')` thing what we used to resolve the bug with the unexpanded uris?